readme.md

```diff
@@ -60,9 +60,12 @@ template:
 ```
 
 
-## Usage with the Subcategory plugin
+## Subcategory plugin support
+
+This plugin is compatible with the [`subcategory`
+plugin](https://github.com/getpelican/pelican-plugins/tree/master/subcategory).
+here is how to use it.
```

Comment:
Maybe change this short sentence to:
> Following below are instructions on how to use Neighbors in conjunction with the Subcategory plugin.

Reply:
Done in: https://github.com/pelican-plugins/neighbors/pull/2/commits/a1eceb2d12dae49ea0a9904a45e5c8bbab0b35ad